Sources/XcbeautifyLib/Parser.swift

```diff
@@ -151,7 +163,12 @@ public class Parser {
         return result.value
     }
 
-    
+    public func summarize(write: (String) -> Void) {
+        if let summary = summary {
+            write(summary.format(with: renderer))
+        }
+    }
```

Comment:
For my understanding, what does this method provide for API consumers that `write(_:)` doesn't?

Reply:
Addressed in a different PR and this implementation approach is no longer relevant.